### PR TITLE
Add a sample for gapi.client.init with loaded discovery.

### PR DESCRIPTION
**Pull request by bochunz**
@bsittler @wora PTAL

### CODE REVIEW

File: samples/loadedDiscovery.html [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+        // Load the API client and auth2 library.
+        var loadGapiClient = new Promise(function(resolve, reject) {
+          gapi.load('client:auth2', resolve);
+        });
+
+        // Fetch the People API discovery served on local server. You can also
+        // pack the json as a string to avoid extra network request.
+        // After this promise is fulfilled, the peopleApiDiscovery variable
+        // will be set.
+        var fetchPeopleApiDiscovery = fetch('people/people_rest_v1.json').then(
```
Comment (bsittler) [inline samples/loadedDiscovery.html]: Please include the Discovery Service URL used to fetch the original copy of this JSON file.

Comment (bochunz) [inline samples/loadedDiscovery.html]: Done

File: samples/loadedDiscovery.html [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+    <!--Add buttons to initiate auth sequence and sign out-->
+    <button id="authorize-button" style="display: none;">Authorize</button>
+    <button id="signout-button" style="display: none;">Sign Out</button>
+
+    <div id="content"></div>
+
+    <script type="text/javascript">
+      // Enter an API key from the Google API Console:
+      //   https://console.developers.google.com/apis/credentials
+      var apiKey = 'YOUR_API_KEY';
```
Comment (bochunz) [inline samples/loadedDiscovery.html]: Very nice idea!

I took a look at the pages served on github.io, you can already load those pages (don't actually need to move them to /docs). e.g. https://google.github.io/google-api-javascript-client/samples/authSample

Comment (bochunz) [inline samples/loadedDiscovery.html]: Will do in a separate PR.
